src/System.Runtime.Analyzers/VisualBasic/System.Runtime.VisualBasic.Analyzers.vbproj

```diff
@@ -116,8 +116,8 @@
     <Compile Include="BasicRethrowToPreserveStackDetails.Fixer.vb" />
     <Compile Include="BasicDoNotRaiseReservedExceptionTypes.vb" />
     <Compile Include="BasicDoNotRaiseReservedExceptionTypes.Fixer.vb" />
-    <Compile Include="BasicInitializeValueTypeStaticFieldsInline.vb" />
-    <Compile Include="BasicInitializeValueTypeStaticFieldsInline.Fixer.vb" />
+    <Compile Include="BasicInitializeStaticFieldsInline.vb" />
```

Comment:
@lgolding @genlu  we need a nice text naming convention defined as well. It is pleasant when the test names map to the rule names (which map to the rule file names)